tests/composer/edition_plugin.test.ts

```diff
@@ -1193,4 +1196,52 @@ describe("edition", () => {
     expect(highlights).toHaveLength(1);
     expect(highlights[0].zone).toMatchObject(toZone("A2"));
   });
+
+  describe("Data validation autocomplete dropdown values", () => {
+    let model: Model;
+    beforeEach(async () => {
+      model = new Model();
+      await mountSpreadsheet({ model });
+    });
+
+    test("Duplicate values will be removed before sending proposals to the autocomplete dropdown in data validation with list", async () => {
```

Comment:
Move these tests witht he other tests of the autocomplete for data validation (tests/data_validation/data_validation_list.test.ts). And use the `typeInComposer` helper rather than the triggerMouseEvent